cluster/charts/crossplane/templates/rbac-manager-managed-clusterroles.yaml

```diff
@@ -26,6 +26,8 @@ aggregationRule:
   clusterRoleSelectors:
   - matchLabels:
       rbac.crossplane.io/aggregate-to-admin: "true"
+      rbac.crossplane.io/aggregate-to-edit: "true"
```

Comment:
Thank a lot for quick response @hasheddan 👍

However, I was more expecting something like https://github.com/crossplane/crossplane/blob/release-0.11/cluster/charts/crossplane/templates/environment-personas-clusterroles.yaml#L119

See, how it bubbles up from view -> edit -> admin

Reply:
Ah, I see. I think this would have the same effect (minus view roles going to edit), but the referenced model feel more composable :+1: I am going to leave as-is for the moment and wait for @negz to weigh in on the general pattern.